bootstrap-salt.sh

```diff
@@ -2887,6 +2886,27 @@ install_ubuntu_check_services() {
 #
 #   Debian Install Functions
 #
+__install_saltstack_debian_repository() {
+    if [ "$DISTRO_MAJOR_VERSION" -eq 7 ]; then
+        DEBIAN_CODENAME="wheezy"
+    else
+        DEBIAN_CODENAME="jessie"
+    fi
```

Comment:
Yeah, I agree. I thought of this, too, but I think a separate PR will be better for this right now. I'll make an issue for it, especially since 7 is phasing and we'll be moving to 9 very soon as you said. :)